app/grandchallenge/challenges/forms.py

```diff
@@ -179,3 +190,265 @@ class Meta:
                 reverse_lazy("publications:create"),
             )
         }
+
+
+general_information_items = (
+    "title",
+    "challenge_short_name",
+    "contact_email",
+    "abstract",
+    "start_date",
+    "end_date",
+    "organizers",
+    "affiliated_event",
+    "structured_challenge_submission_form",
+    "task_types",
+    "structures",
+    "modalities",
+    "challenge_type",
+    "challenge_setup",
+    "data_set",
+    "submission_assessment",
+    "challenge_publication",
+    "code_availability",
+    "expected_number_of_teams",
+)
+phase_1_items = (
+    "phase_1_number_of_submissions_per_team",
+    "phase_1_number_of_test_images",
+)
+phase_2_items = (
+    "phase_2_number_of_submissions_per_team",
+    "phase_2_number_of_test_images",
+)
+
+
+class ChallengeRequestForm(forms.ModelForm):
+    class Meta:
+        model = ChallengeRequest
+        fields = (
+            *general_information_items,
+            "number_of_tasks",
+            "average_size_of_test_image",
+            "inference_time_limit",
+            *phase_1_items,
+            *phase_2_items,
+        )
+        widgets = {
+            "start_date": forms.TextInput(attrs={"type": "date"}),
+            "end_date": forms.TextInput(attrs={"type": "date"}),
+            "challenge_type": forms.Select(
+                attrs={"onchange": "updateForm();"}
+            ),
+        }
+        help_texts = {
+            "challenge_short_name": "Short name that will be used in the URL "
```

Comment:
This is quite difficult to parse and find the other keys, could you wrap the help text in brackets? eg:

```
"challenge_short_name": (
    "Short name that will be used in the URL "
    ...
)
```